exporters/geneva/src/exporter.cc

```diff
@@ -232,14 +267,18 @@ size_t Exporter::SerializeNonHistogramMetrics(
           .count());
 
   SerializeInt<uint64_t>(buffer_non_histogram_, bufferIndex, windows_ticks);
-  if (event_type == MetricsEventType::ULongMetric) {
+  if (event_type == MetricsEventType::Uint64Metric) {
     SerializeInt<uint64_t>(buffer_non_histogram_, bufferIndex,
                            static_cast<uint64_t>(nostd::get<int64_t>(value)));
-  } else {
+  } else if (event_type == MetricsEventType::DoubleMetric)
+  {
```

Comment:
nit: join this with above line?

Reply:
thanks fixed. Also fixed clang formatting for the code in general